app/src/main/java/org/coepi/android/di/Modules.kt

```diff
@@ -25,7 +25,7 @@ import org.koin.dsl.module
 
 val viewModelModule = module {
     viewModel { HomeViewModel(get()) }
-    viewModel { SymptomsViewModel(get()) }
+    viewModel { SymptomsViewModel(get(),get()) }
```

Comment:
Please use autoformat  https://jimbaca.com/auto-format-code-in-android-studio/

Reply:
added 1 space
 -    viewModel { SymptomsViewModel(get(),get()) }
+    viewModel { SymptomsViewModel(get(), get()) }